### PR TITLE
Fixed more golint stuff 

### PR DESCRIPTION
**Pull request by ivenk**
Fixes parts of #18 

Fixed most issues in plugin.go.

### CODE REVIEW

File: server/plugin.go [hunk 1]
```diff
@@ -150,13 +150,13 @@ func (p *Plugin) completeAuthorizeInADD(w http.ResponseWriter, r *http.Request)
...
 
-	userId, err := p.API.KVGet(state)
+	userID, err := p.API.KVGet(state)
 
 	if err != nil {
 		fmt.Println(err.Message)
 		http.Error(w, "cannot get stored state", http.StatusBadRequest)
 		return
-	} else if userId == nil {
+	} else if userID == nil {
```
Comment (hanzei) [inline server/plugin.go]: you can drop the `else` here. Would you mind applying this change to the whole file?

File: server/plugin.go [hunk 2]
```diff
@@ -538,28 +540,28 @@ func (p *Plugin) getApplicationState(discoveryUrl string) (*ApplicationState, *A
...
 	}
 
-	rootUrl, err := p.determineRootUrl(p.getConfiguration().Domain)
+	rootURL, err := p.determineRootURL(p.getConfiguration().Domain)
 	if err != nil {
 		return nil, err
 	}
 
-	_ = p.API.KVSet(ROOT_URL_KEY, []byte(*rootUrl))
+	_ = p.API.KVSet(RootURLKey, []byte(*rootURL))
```
Comment (hanzei) [inline server/plugin.go]: Can you please check the error here?

File: server/plugin.go (hunk 1 above)
Comment (ivenk) [reply server/plugin.go]: Will do.

File: server/plugin.go (hunk 2 above)
Comment (ivenk) [reply server/plugin.go]: Will do.

**Comment by hanzei** [comment]
@ivenk Would you please merge `master` into your branch?

**Comment by ivenk** [comment]
will do. maybe tomorrow.

**Comment by ivenk** [comment]
Okay so i dont have the repository locally anymore. Instead i applied the changes done in this pr to the up to date version.

**Comment by hanzei** [comment]
All fine. Will close this PR

**Pull request closed without merging**